diagnostic.md

```diff
@@ -31,25 +30,21 @@ the `form` DOM node. (Assume the list item has a class of "listerItem").
 Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
-
+.val() getter
+.text() getter
+.val("hello") setter
+.text('hello') setter
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
-
+The Dom is a visual representaion of the source code we send to the browser
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.  A web browser implements this interface to render web pages for you to view and interact with.